widgets/Snowflake-batchsource.json

```diff
@@ -36,6 +36,30 @@
           "label": "Role",
           "name": "role"
         },
+        {
+          "name": "importQueryType",
+          "label": "Import Query Type",
+          "widget-type": "radio-group",
+          "widget-attributes": {
+            "layout": "inline",
+            "default": "importQuery",
```

Comment:
While getting the schema, we're checking for the `tableName` first and then the `importQuery`, so shouldn't `tableName` be the default value for the `importQueryType` ?